docs/testimonials.md

```diff
@@ -0,0 +1,118 @@
+# Testimonials
+
+You might not be ready to make the jump to Jujutsu yet. It's understandable; new
+tools come with new lessons, failures, and ideas to absorb. They require
+practice. In order to provide some motivation, we've collected a number of real,
+100% authentic testimonials &mdash; from our loving users, our silly developers
+&mdash; all to tip the scales and get you on our side!
+
+## What the users have to say 
+
+> I've spent many years of my career working on version control. What I like
+> most about Jujutsu is how it has non-obvious solutions to UX problems that
+> we've run into in the past. What most people may not realize is that there are
+> many novel features which all interlock to make it easy to use.
+>
+> For example, consider Jujutsu's support for automatically rebasing descendants
+> of amended revisions. When we implemented that in Mercurial, we ran into an
+> issue: what if there's a merge conflict? Our solution was to warn users and
+> just not perform the auto-rebase. Now, suddenly, users have to understand that
+> there can be old versions of the same revision visible in their log, and learn
+> how to fix this state.
+>
+> In contrast, Jujutsu's solution is to simply make merge conflicts first-class.
+> This is not just an improvement in general, it is also specifically an
+> improvement for auto-rebase &mdash; users no longer have to learn about old
```

Comment:
Interestingly, most style guides say that you [shouldn't put spaces around em-dashes](https://prowritingaid.com/grammar/1000066/Should-I-use-spaces-around-an-em-dash). Like many people I've been in the habit of putting spaces around em-dashes, though I've been trying to move away from that.

Either way's fine and consistency is most important, but just wanted to mention that.